apps/api/src/database/schema.prisma

```diff
@@ -570,7 +570,7 @@ model RepresentationAction {
   id                   Int            @id @default(autoincrement())
   representationId     Int
   type                 String? // STATUS | REDACTION | REDACT_STATUS
-  status               String?
+  status               String? // AWAITING_REVIEW | REFERRED | INVALID | VALID | WITHDRAWN
```

Comment:
Just remove as requested before.

the validator has the array of statues and this is missing 3 of the other statues.

Reply:
sorry, I thought that was gone, removed